packages/components/src/core/Dropdown/index.tsx

```diff
@@ -221,7 +221,9 @@ const Dropdown = <Multiple extends boolean | undefined = false>({
       }
     } else {
       if (multiple) {
-        setPendingValue(value as MUIValue<true>);
```

Comment:
I think we shouldn't change this one cause it's a `SetPendingValue` and not a `SetValue`!

Reply:
Ooh yes thanks so much for catching this critical bug 😆 Really appreciate it! Reverted 🙆‍♂️